lib/ansible_tower_client/job_template.rb

```diff
@@ -3,8 +3,15 @@ class JobTemplate
     extend CollectionMethods
     include InstanceMethods
 
-    def launch
-      Api.post("#{url}launch/")
+    attr_reader :response
+
+    def launch(vars = {})
+      launch_url = "#{url}launch/"
+      @response = Api.post(launch_url, vars).body
```

Comment:
Yes - I like that idea.  I sort of forgot that we could actually build a real class around that response. I'll get that added.